imap/cyr_withlock_run.c

```diff
@@ -0,0 +1,138 @@
+/* cyr_pwd.c -- current working directory within a spool dir
```

Comment:
whoops ;)

Reply:
This looks like it still needs to be fixed

Reply:
what?  I swear I fixed that